app/views/categories/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container-fluid catag">
+  <div class="row ">
+    <% @categories.each do |category| %>
+      <div class="row catago" onclick="window.location.href='<%= category.expenses.present? ? expense_path(category.expenses.first) : expenses_path(category_id: category.id) %>';">
+        <div class="col">
+          <div class="image-container">
+          <% if category.icon.attached? %>
+          <%= image_tag category.icon, class: "category-icon img-fluid" %>
+        <% else %>
+          <p>No Image</p>
+        <% end %>
+          </div>
+        </div>
+
+        <div class="col">
+          <p><%= category.name %></p>
+          <% created_date = category.created_at.strftime('%e%p %B %Y').strip %>
+          <% formatted_date = created_date.sub(/\d+(st|nd|rd|th)/, '\1') %>
+          <p><%= formatted_date %></p>
+        </div>
+
+        <div class="col">
+          <% expense = category.expenses.first %>
+          <% if expense %>
+            <p><%= number_to_currency(expense.amount, unit: '$') %></p>
+          <% else %>
+            <p>No Expense</p>
+          <% end %>
+        </div>
+      </div>
```

Comment:
- [ ] It is a requirement because the last code reviewer cannot access the category page and other pages. After adding some expenses in the Food category for an amount of $40 it should be presented in the category#index page, but it shows just the first expense.

![image](https://github.com/danutibe07/BudgetApp/assets/58642949/6a61141f-2f5d-4444-8a2a-ff3ab07b2515)

- [ ] Please correct the styles of the Transactions page to be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://github.com/danutibe07/BudgetApp/assets/58642949/61783892-3cdc-46d1-8cea-a253f93ec102) | ![image](https://github.com/danutibe07/BudgetApp/assets/58642949/022b60d6-7964-4b24-ba2c-655a47f4d0ea) |